release/release.js

```diff
@@ -0,0 +1,332 @@
+  await readInFile('./webpack.config.babel.js',
+    /'BASE_32_VERSION'\s*: '([A-Z]+)'/,
+    (matchGroup) => {
+      console.log(`${chalk.bold.yellow(`${config.data.UPLOAD_CMD} ${config.data.S3_BUCKET}${config.data.BASE_32_FOLDER_PATH}/${matchGroup[1]}/`)}`)
+      const versionPath = config.data.versionRC ? config.data.versionRC : VERSION
+      console.log(`${chalk.bold.yellow(`${config.data.UPLOAD_CMD} ${config.data.S3_BUCKET}${config.data.RELEASES_FOLDER_PATH}/${versionPath}/`)}`)
+    }
+  )
+  await new Promise(resolve => setTimeout(resolve, 1000))
+}
+
+const didS3uploadSucceed = async () => {
+  await proceedYesNo('Have all of these commands succeeded?')
+}
+
+const publishTag = async () => {
+  if (config.data.versionRC) {
+    stepTitle(`🕑 Creating next tag for release candidate ${config.data.versionRC}`)
+    await spawnAssumeOkay('npm', ['publish', '--tag', 'next'])
+    console.log('Done. Now make sure that the latest tag has not changed, only the next one:')
+
+    const isVerboseCmd = true
+    await spawnAssumeOkay('npm', ['dist-tag', 'ls', 'onfido-sdk-ui'], isVerboseCmd)
+    await proceedYesNo('Is it all good?')
+  }
+  else {
+    stepTitle(`🕑 Creating tag ${VERSION}`)
+    await spawnAssumeOkay('git', ['tag', VERSION])
+    await spawnAssumeOkay('git', ['push', 'origin', VERSION])
+    console.log(`Done. The latest tag should now be ${VERSION}`)
+    console.log(`Now check that: `)
+    console.log('- Travis TAG build was successfull')
+    console.log(`- https://latest-onfido-sdk-ui-onfido.surge.sh/ is using ${VERSION}`)
+    await proceedYesNo('Is it all good?')
+  }
+}
+
+const checkNPMUserIsLoggedIn = async () => {
+  const isLoggedIn = await execWithErrorHandling('npm whoami', npmLoginInstruction)
+  if (isLoggedIn) {
+    console.log('✅ Success!')
+  }
+}
+
+const npmLoginInstruction = async () => {
+  console.log('Oops! Looks like you are not logged in.')
+  console.log(`In a new tab, run ${chalk.bold.yellow('npm login')} using the credentials from 1Password`)
+  await proceedYesNo('All good?')
+  await checkNPMUserIsLoggedIn()
+}
+
+const npmLogin = async () => {
+  stepTitle(`🔑 NPM login`)
+  await checkNPMUserIsLoggedIn()
+}
+
+const publishOnNpm = async () => {
+  stepTitle(`🚀 Publishing ${VERSION} on NPM`)
+  await spawnAssumeOkay('npm', ['publish'])
+  console.log('✅ Success!')
+}
+
+const upgradeDemoAppToTag = async () => {
+  stepTitle('🕑 Upgrading demo app...')
+  const versionToInstall = config.data.versionRC ? config.data.versionRC : VERSION
+  await spawnAssumeOkay(`cd ${config.data.SAMPLE_APP_PATH} && npm install onfido-sdk-ui@${versionToInstall}`, [])
+  console.log('✅ Success!')
+}
+
+const regressionTesting = async () => {
+  stepTitle('👀 Regression testing')
+  console.log('✅ Release candidate complete!')
+  console.log('🏃 Go ahead and test the SDK deployment on surge link associated with the PR')
+  console.log('Note: Use https://release-[PR-NUMBER]-pr-onfido-sdk-ui-onfido.surge.sh/')
```

Comment:
Probably can just combine the Surge PR link with the line above. Or re-word line 228 to point to the link in the next line.